scripts/prepare_statement.py

```diff
@@ -2,14 +2,15 @@
 import json
 import argparse
 
-def prepare(circuit_file, output_file, name, statement_type):
+
+def prepare(circuit_file, output_file, name, statement_type, private):
     data = {
         "name": name,
         "description": "description",
         "url": "url",
         "input_description": "input_description",
         "type": statement_type,
-        "isPrivate": False,
+        "isPrivate": private,
```

Comment:
It is boolean field, so i will be better to set "true"

Reply:
What do you mean? `private` is an argument of the `prepare()` function that gets boolean values.